docs/dashboard/dimensions-contexts-families.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: "Chart dimensions, contexts, and families"
+description: "Netdata organizes charts into dimensions, contexts, and families to automatically and meaningfully organize thousands of metrics into interactive charts."
+type: explanation
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/dashboard/dimensions-contexts-families.mdx
+---
+
+# Chart dimensions, contexts, and families
+
+While Netdata's charts require no configuration and are [easy to interact with](/docs/dashboards/interact-charts.mdx),
+they have a lot of underlying complexity. To meaningfully organize charts out of the box based on what's happening in
+your nodes, Netdata uses the concepts of **dimensions**, **contexts**, and **families**. 
+
+Understanding how these work will help you more easily navigate the dashboard, [write new
+alarms](/docs/monitor/configure-alars.md), or play around with the [API](/web/api/README.md).
+
+For a refresher on the anatomy of a chart, see [dashboards and charts](/docs/dashboards/dashboards-charts.mdx).
+
+## Dimension
+
+A **dimension** is a value that gets shown on a chart. The value can be raw data or calculated values, such as
+percentages, aggregates, and more.
```

Comment:
I am not sure if `aggregates` are given the expected result, because we can aggregate data with different functions, can we add a link for the list of aggregated methods available?

Reply:
Do you know where a list like that might be?

Reply:
The simplest doc that I found talking about our aggregated methods, or data grouping is defined in this [link](https://github.com/netdata/netdata/tree/master/web/gui/custom#data-grouping-method).

Reply:
Worked on this a bit. I didn't add the link, since it seems like we don't offer a complete picture anywhere, but I think this new version is clearer. LMK!

Reply:
I liked your changes, I review one more time, before to approve the doc. Thanks!